### DISCUSSION TITLE
UI load up state

### DISCUSSION THREAD

**Discussion by SGThirkell** [lllyasviel/Fooocus#2451] [Ideas]
Hi, would there be any way of adding the feature of "remembering" the state of open menus/check boxes ticked, so that when Fooocus is started up, it goes into the same state it was left at previously, without having to check boxes, open menus etc from scratch again?

**Comment by mashb1t** [comment]
This is not trivial and also not possible out of the box. Feel free to implement it and create a PR!

Relates to https://github.com/lllyasviel/Fooocus/issues/1622

**Comment by blckbx** [comment]
You could load parameters of the last generated image from history log. Besides debug checkbox, I think every setting is restored this way. Saves me some clicks.

**Reply by mashb1t** [reply]
true, but still doesn't open the UI elements as they were opened before
